test/glue.js

```diff
@@ -0,0 +1,85 @@
+const tape = require('tape')
+const ssbKeys = require('ssb-keys')
+const Server = require('scuttle-testbot')
+const u = require('./util')
+
+const followedKeys = ssbKeys.generate()
+const blockedKeys = ssbKeys.generate()
+
+tape('listen to hopStream and requests replication of follows', function (t) {
+  t.pass('followed feed is ' + followedKeys.id)
+
+  const sbot = Server
+    .use({
+      name: 'replicate',
+      init() {
+        return {
+          request(feed, bool) {
+            if (feed === sbot.id) return
+            t.equals(feed, followedKeys.id, 'requested feed id matches')
+            t.true(bool, 'bool is true')
+            sbot.close(err => {
+              t.error(err, 'close sbot')
+              t.end()
+            })
+          }
+        }
+      }
+    })
+    .use(require('..'))
+    .call(null, {
+      friends: {
+        hops: 2
+      }
+    })
+
+  sbot.friends.follow(followedKeys.id, {}, (err, msg) => {
+    t.error(err, 'no error when following')
+    t.ok(msg, 'contact msg is truthy')
+  })
+})
+
+tape('listen to hopStream and stops replication of blocks', function (t) {
```

Comment:
Mocks the replicate plugin to check that `sbot.replicate.block` is called, both when blocking or unblocking.